packages/react-search-ui/package.json

```diff
@@ -3,16 +3,15 @@
   "version": "1.9.0",
   "description": "A React library for building search experiences",
   "license": "Apache-2.0",
-  "main": "lib",
```

Comment:
@JasonStoltz you spotted this in previous PR - fixed here. Found the issue when I started to use the package!